src/main/java/ca/bc/gov/backendstartapi/config/SwaggerConfig.java

```diff
@@ -51,6 +54,27 @@ public OpenAPI theRestApi() {
                         .type(Type.HTTP)
                         .scheme("bearer")
                         .bearerFormat("JWT")))
-        .addSecurityItem(new SecurityRequirement().addList(BEARER_SECURITY_SCHEME_NAME));
+        .addSecurityItem(new SecurityRequirement().addList(BEARER_SECURITY_SCHEME_NAME))
+        // TODO: find a way to keep this closer to the controller itself.
+        .paths(
+            new Paths()
+                .addPathItem(
+                    "/api/seedlot-status",
```

Comment:
Paths are discovered automatically by spring-doc. Don't need to register them in the code.